core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -356,9 +359,43 @@ public String toString() {
           .setIsHidden(true)
           .setScope(Scope.ALL)
           .build();
+  public static final PropertyKey NETWORK_CONNECTION_AUTH_TIMEOUT =
```

Comment:
@ggezer  are you good with the following?

Reply:
These LGTM. Thanks!